services/roomService/src/types/Game.test.ts

```diff
@@ -1,97 +1,187 @@
-// import { nanoid } from "nanoid";
-// import pool from "../dbconnector/pool";
-// import { deleteMazeCompletionTime, getMazeCompletionTime, selectPlayerCompletionTime } from "../utils/queries";
-// import Player from "./Player";
-// import { MazeCompletionTimeRow } from "../requestHandlers/CoveyTownRequestHandlers"
+import { nanoid } from 'nanoid';
+import { MazeCompletionTimeList } from '../CoveyTypes';
+import pool from '../dbconnector/pool';
+import CoveyTownController from '../lib/CoveyTownController';
+import { MazeCompletionTimeRow } from '../requestHandlers/CoveyTownRequestHandlers';
+import CoveyTownListener from '../types/CoveyTownListener';
+import { deleteMazeCompletionTime, getMazeCompletionTime } from '../utils/queries';
+import Player from './Player';
 
-describe('dummy test', () => {
-  it('test to make this pass for now', () => {
-    expect(1).toEqual(1);
+const mockCoveyListenerTownDestroyed = jest.fn();
```

Comment:
Since we don't do any specific testing on town destroyed, this should be replaceable with just mockCoveyListenersOtherFns